data/sql/updates/pending_db_world/repandspawn.sql

```diff
@@ -0,0 +1,4 @@
+--lots of Reputation changes
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepFaction1`=87, `RewOnKillRepFaction2`=21, `MaxStanding1`=5, `IsTeamAward1`=0, `RewOnKillRepValue1`=5, `MaxStanding2`=7, `IsTeamAward2`=1, `RewOnKillRepValue2`=-25, `TeamDependent`=0 WHERE `creature_id`=26081;
+--Spawn to 8 hours
```

Comment:
```suggestion
-- Spawn to 8 hours
```

Reply:
now the purpose of spacing after -- is for the ci runs. --XX and -- XX is viewed differently with the workflow.